backend/packages/Upgrade/src/api/controllers/ExperimentClientController.ts

```diff
@@ -292,13 +292,27 @@ export class ExperimentClientController {
    *            description: Log blob data
    */
   @Post('bloblog')
-  public blobLog(@Req() request: express.Request): Promise<Log[]> {
-    return new Promise((resolve) => {
+  public blobLog(@Req() request: express.Request): any {
+    return new Promise((resolve, reject) => {
       request.on('readable', async (data) => {
         const blobData = JSON.parse(request.read());
-        const response = await this.experimentAssignmentService.blobDataLog(blobData.userId, blobData.value);
-        resolve(response);
-      });
+        try {
+          // The function will throw error is iserId doesn't exist
```

Comment:
Can you please update the comment by fixing the typo?

Reply:
Sorry about this, will fix it.

Reply:
thanks!